c10/core/TensorImpl.cpp

```diff
@@ -669,7 +678,7 @@ void TensorImpl::Extend(int64_t num, float growthPct) {
           sizes_and_strides_.size_at_unchecked(0).as_int_unchecked() *
           (1 + growthPct / 100))));
   auto oldData = std::move(storage_.data_ptr());
-  auto oldSize = numel_;
+  auto oldSize = numel_.expect_int();
```

Comment:
This can be unchecked, as you have already tested symbolic sizes strides field above